.github/workflows/secrel4prs.yml

```diff
@@ -64,26 +64,44 @@ jobs:
           ghcr_password: ${{ secrets.GITHUB_TOKEN }}
           run_tests: ${{ env.RUN_TESTS }}
 
-  debug-job:
+  images-for-secrel:
     runs-on: ubuntu-latest
     needs:
       - publish-images
+    outputs:
+      all-images: ${{ steps.append-images.outputs.all-images }}
+    steps:
+      - name: "Append 3rd-party images"
+        id: append-images
+        run: |
+          echo "all-images<<EOF" >> $GITHUB_OUTPUT
+          echo "${{ needs.publish-images.outputs.vro-images }}" >> $GITHUB_OUTPUT
+          MANUAL_IMGS_ARR=( vro-rabbitmq:3 vro-redis:7 )
+          for MANUAL_IMAGE in ${MANUAL_IMGS_ARR[@]}; do
+            echo "ghcr.io/department-of-veterans-affairs/abd-vro-internal/${MANUAL_IMAGE}" >> $GITHUB_OUTPUT
```

Comment:
[try abd-vro-internal for 3rd-party images](https://github.com/department-of-veterans-affairs/abd-vro/pull/963/commits/61ac25bc8514971bdb5274f52ab6182a4096ea13) to fix https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/4001093791/jobs/6867068970#step:4:26

Reply:
Confirmed that SecRel cannot access package from the public repo. Odd, I'll ask them why.